src/Microsoft.TestPlatform.ObjectModel/TestAdapterLoadingStrategy.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel
+{
+    /// <summary>
+    /// Represents a loading strategy
+    /// </summary>
+    [Flags]
+    internal enum TestAdapterLoadingStrategy
+    {
+        /// <summary>
+        /// A strategy not defined, Test Platfrom will load adapters normally. 
+        /// </summary>
+        Default = 0b0000_0000_0000_0000,
+
+        /// <summary>
+        /// Test Plarform will only load adapters specified by /TestAdapterPath (or RunConfiguration.TestAdaptersPaths node). 
+        /// If a specific adapter path is provided, adapter will be loaded; if a directory path is provided adapters directly in that folder will be loaded. 
+        /// If no adapter path is specified, test run will fail.
+        /// This will imply /InIsolation switch and force the tests to be run in an isolated process.
```

Comment:
```suggestion
        /// Test Platform will only load adapters specified by /TestAdapterPath (or RunConfiguration.TestAdaptersPaths node). 
        /// If a specific adapter path is provided, adapter will be loaded; if a directory path is provided adapters directly in that folder will be loaded. 
        /// If no adapter path is specified, test run will fail.
        /// This will imply /InIsolation switch and force the tests to be run in an isolated process.
```